frontend/tests/features/cellCards/cellCards.test.ts

```diff
@@ -71,13 +71,14 @@ describe("Cell Guide", () => {
       page,
     }) => {
       await goToPage(`${TEST_URL}${ROUTES.CELL_GUIDE}`, page);
-      const element = page.getByTestId(CELL_GUIDE_CARD_SEARCH_BAR_TEXT_INPUT);
+
+      const element = getSearchBarLocator(page);
```

Comment:
use `getSearchBarLocator()`, so we get the actual `<input />` field instead of the `<div />`, so `element.clear()` will work